### ISSUE TITLE
Searching for 'github_data/dmpl_sample.npz'

### ISSUE THREAD

**Issue by skore11** [nghorbani/human_body_prior#57]
Hello,
I am working on bringing DMPL to a real time collision like seen in Unity.
Are the contents of the dmpl_sample.npz file from the DMPL visualization .ipynb notebook the Dyna registrations?
Meaning you need the eig vectors of the errors between the vertices of SMPL and Dyna correct?
Also I did not see much of a difference between the animation with DMPLs and without. Is this because the number of DMPL blendshape principal components provided by SMPL are not enough?
Keep up the good work!
Thanks
Abishek

**Comment by MichaelJBlack** [comment]
Are you getting the SMPL/DMPL parameters from AMASS?  I only recently realized that the released AMASS fits are fits to virtual markers using MoSh and not fits to the scan data.  The former seems not very good, while the latter is very detailed.  I'm looking into it.

**Comment by MichaelJBlack** [comment]
Right.  With sparse markers, you can't get much soft tissue detail, hence, we use few DMPLs.  I'm working on getting the more detailed Dyna/DFAUST fits on the AMASS site.

**Comment by skore11** [comment]
That sounds good, Ill keep a lookout for the same, on a side note do you know if DMPLs will be included in the Unity build? I am trying to include the same with position based dynamics to integrate collision based blendshapes with SMPL and DMPL. I guess the big issue would be to redo lbs with the DMPL blendshapes.
